lighthouse-core/report/html/report-template.html

```diff
@@ -20,7 +20,9 @@
 
   <div id="lh-log"></div>
 
-  <script>%%LIGHTHOUSE_JAVASCRIPT%%</script>
+  <script>%%LIGHTHOUSE_JAVASCRIPT%%
+  //# sourceURL=report-renderer-js.js
```

Comment:
does this work with anything?

Reply:
it enables a few things in devtools. like code coverage some other features that are wonky with inline `<script>` tags.
you can also just ctrl-p into the JS rather than jump to some line in the big huge HTML.